packages/vueform/stories/VaSelectElement.stories.ts

```diff
@@ -0,0 +1,29 @@
+import { StoryFn } from '@storybook/vue3'
+
+export default {
+  title: 'Vueform Integration/Select',
+}
+
+export const Default: StoryFn = () => ({
+  setup() {
+    const frameworks = [
+      { text: 'React', value: 'react' },
+      { text: 'Vue', value: 'vue' },
+      { text: 'Angular', value: 'angular' },
+      { text: 'Svelte', value: 'svelte' },
+    ]
+    return {
+      frameworks,
+    }
+  },
+  template: `
+  <Vueform>
+      <VaSelectElement
+        name="framework"
+        searchable
```

Comment:
For stories, what was the choice of props?

To make it similar to vueform examples?